api/models/teacher.py

```diff
@@ -115,3 +115,7 @@ class Meta:
     def can_take_more_groups(self) -> bool:
         """`True` if a teacher can take more groups than they already have."""
         return self.groups.count() < self.simultaneous_groups
+
+    @property
+    def has_groups(self) -> bool:
+        return self.groups.count() > 0
```

Comment:
Note that I count all groups but maybe we should count only active groups? (same for student model)

Reply:
Same as above